### PR TITLE
Clean up the classes in pageobjects directory

### PR DESCRIPTION
**Pull request by bartzet**
# Problem
The structure of the classes in the `pageobjects` directory needs improvements.

# Solution
1. Rename the `Common` class to `Screen`
2. `Screen` extends from `Base`
3. Other classes in `pageobjects` extend from `Screen`
4. Move `copy(lang) { return locale(lang) }` to `Screen` class and remove it from other `pageobject` classes.
5. Fix imports.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-753-pr-onfido-sdk-ui-onfido.surge.sh

File: test/pageobjects/CameraPermissions.js [hunk 1]
```diff
@@ -1,16 +1,14 @@
-import Base from './BasePage.js'
-import {locale, verifyElementCopy} from '../utils/mochaw'
+import {verifyElementCopy} from '../utils/mochaw'
+import Screen from './Screen.js'
```
Comment (stefaniacardenas) [inline test/pageobjects/CameraPermissions.js]: Why renaming it to `Screen`? It doesn't give me the idea of something shared between multiple components.

File: test/pageobjects/Screen.js [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
+class Screen extends Base {
   get backArrow() { return this.$('.onfido-sdk-ui-NavigationBar-iconBack')}
 
-  copy(lang) { return locale(lang) }
-
   async clickBackArrow() {
     this.backArrow.click()
   }
+
+  copy(lang) {
```
Comment (stefaniacardenas) [inline test/pageobjects/Screen.js]: Nice to get rid of all that repetition 👍

File: test/pageobjects/CameraPermissions.js (hunk 1 above)
Comment (bartzet) [reply test/pageobjects/CameraPermissions.js]: `Screen` doesn't represent a particular screen but something higher-level that other test classes can inherit from.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://753-pr-onfido-sdk-ui-onfido.surge.sh

File: test/specs/happy.js [hunk 3]
```diff
@@ -639,20 +639,20 @@ describe('Happy Paths', options, ({driver, pageObjects}) => {
...
 
       it('should navigate to the second-last step of the flow and then go back to the beginning', async () => {
-        const copy = common.copy(lang)
+        const copy = screen.copy(lang)
         goToPassportUploadScreen(`?language=${lang}&async=false&useWebcam=false`)
         uploadFileAndClickConfirmButton('passport.jpg')
         documentUpload.getUploadInput()
         documentUpload.upload('face.jpeg')
-        common.clickBackArrow()
+        screen.clickBackArrow()
```
Comment (rfreitas) [inline test/specs/happy.js]: hm, it would make more sense to call `clickBackArrow` on the screen we assume the user is in

Comment (bartzet) [reply test/specs/happy.js]: In that case 

```
async clickBackArrow() {
    this.backArrow.click()
}
```
would have to be in every page object class. If Screen class is not for the selectors that repeat across the different screens it makes sense to move it. 

So for now, Screen would only be for:

```copy(lang) {
    return locale(lang)}

Comment (rfreitas) [reply test/specs/happy.js]: >  If Screen class is not for the selectors that repeat across the different screens it makes sense to move it.
I don't follow this statement.

all page objects that you press back on are screens, so they all should inherit from the super call screen, which should have the method to click back

Comment (rfreitas) [reply test/specs/happy.js]: screen shouldn't really be treated as instance, but more like an abstract class, which is to say a class that a concrete will inherit from.

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-753-pr-onfido-sdk-ui-onfido.surge.sh

File: test/specs/happy.js (hunk 3 above)
Comment (bartzet) [reply test/specs/happy.js]: Thanks for explanation. This is now done in 38cf58a

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-753-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by bartzet** [comment]
Closing this PR and will open a new one on top of the development to avoid merge conflicts after the big refactoring was done.

**Pull request closed without merging**